app/uk/gov/hmrc/incometaxsessiondata/controllers/SessionController.scala

```diff
@@ -32,31 +34,47 @@ class SessionController @Inject()(cc: ControllerComponents,
                                   authentication: AuthenticationPredicate,
                                   sessionService: SessionService
                                  )(implicit ec: ExecutionContext)
-    extends BackendController(cc) with Logging {
+  extends BackendController(cc) with Logging {
 
-  def getById(sessionID: String): Action[AnyContent] = authentication.async {  request =>
+  def get(mtditid: String): Action[AnyContent] = authentication.async { request =>
     // Here is required internalID => request.internalId and request.sessionId
-    sessionService.get(sessionID) map {
+    sessionService.get(request.sessionId, request.internalId, mtditid) map {
```

Comment:
you can pass request into sessionService.get as its contains all required params

Reply:
ah yes, I can do this now after merging main :D